perceval/backends/git.py

```diff
@@ -611,6 +612,33 @@ def pull(self):
         logging.debug("Git %s repository pulled into %s",
                       self.uri, self.dirpath)
 
+    def _read_stderr (self):
```

Comment:
As a general rule, I use to put _protected_ methods after _public_ methods and sorted by when they are called.